vlm/src/test/scala/geotrellis/contrib/vlm/gdal/URIFormatterSpec.scala

```diff
@@ -0,0 +1,73 @@
+package geotrellis.contrib.vlm.gdal
+
+import java.net.URI
+
+import org.scalatest._
+
+
+class URIFormatterSpec extends FunSpec with Matchers {
```

Comment:
Can we add more test cases here? https://github.com/mapbox/rasterio/blob/master/tests/test_path.py